HM1task5.c

```diff
@@ -0,0 +1,57 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+bool check_balance(char str[]) 
+{
+    int count_parentheses = 0;
+    int count_brackets = 0;
+    int count_braces = 0;
+    for(int i = 0; str[i] != '\0'; i++)

```

Comment:
```suggestion
    for (int i = 0; str[i] != '\0'; i++)
```
После ключевых слов `if`, `else`, `for`, `while` должен быть пробел